### PR TITLE
cloudreve: Update to 3.8.0

### PR DESCRIPTION
**Pull request by 1715173329**
Maintainer: me
Compile tested: rockchip/armv8
Run tested: nanopi-r2s

Description:
Updated cloudreve and added missing web frontend assets archive.

### CODE REVIEW

File: utils/7z/Makefile [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+PKG_MAINTAINER:=Tianling Shen <cnsztl@immortalwrt.org>
+PKG_LICENSE:=LGPL-2.1-or-later BSD-3-Clause
+PKG_LICENSE_FILES:=DOC/License.txt DOC/copying.txt
+PKG_CPE_ID:=cpe:/a:7-zip:7-zip
+
+HOST_BUILD_PARALLEL:=1
+PKG_BUILD_PARALLEL:=1
+
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/host-build.mk
```
Comment (neheb) [inline utils/7z/Makefile:20]: Why?

Comment (1715173329) [reply utils/7z/Makefile:20]: cloudreve needs it to create zip archive.

Comment (neheb) [reply utils/7z/Makefile:20]: This is a host package already provided by tools/7zip.

Comment (1715173329) [reply utils/7z/Makefile:20]: Yep, but it's 7zr, which can only handle .7z archives.

Comment (neheb) [reply utils/7z/Makefile:20]: If the compilation options for tools/7zip are inadequate, submit a patch to fix those.

Comment (neheb) [reply utils/7z/Makefile:20]: Wait a minute, you're creating a zip file. Why not use libdeflate instead?

Comment (1715173329) [reply utils/7z/Makefile:20]: > Wait a minute, you're creating a zip file. Why not use libdeflate instead?

If I understand correctly libdeflate can only create gzip archive, it's not suitable here.

And I see zip is already in tools https://github.com/openwrt/openwrt/tree/main/tools/zip, I must have missed something

Comment (neheb) [reply utils/7z/Makefile]: Ah right. libdeflate is only for creating gzip files.

File: net/cloudreve/Makefile [hunk 2]
```diff
@@ -35,7 +35,7 @@ define Package/cloudreve
   SUBMENU:=Cloud Manager
   TITLE:=A project helps you build your own cloud in minutes
   URL:=https://cloudreve.org
-  DEPENDS:=@(aarch64||arm||i386||i686||x86_64) +ca-bundle
+  DEPENDS:=@(aarch64||arm||i386||i686||riscv64||x86_64) +ca-bundle
```
Comment (neheb) [inline net/cloudreve/Makefile:38]: Wasn’t there a GO_ARCH_DEPENDS variable or something like that?

Comment (1715173329) [reply net/cloudreve/Makefile:38]: `mips/mips64/mips64el/mipsel/powerpc64` are unsupported by cloudreve, though i can use `@!(...)` but this line would be even longer

**Pull request merged**